fs/fs.go

```diff
@@ -57,12 +66,13 @@ func (fs *Fs) ProjectFilepaths() ([]string, error) {
 		if item.IsDir() {
 			continue
 		}
+
 		filepaths = append(filepaths, filepath.Join(dir, item.Name()))
 	}
+	filteredFilepaths := fs.filterBackupFiles(filepaths)
```

Comment:
Would it make sense to directly check in the preceding `for` loop whether the item name has a `.bak` suffix? This would improve the runtime performance.

Reply:
Yes, that is possible. I think performance shouldn't be affected too much since we'd be bounded by O(n) for both for loops though. 
My reasoning for making  a helper function for this was re-usability, in the case we may need to filter out the backup files elsewhere, and to limit side effects in the preceding for loop, however, I think I will make the change and we can revisit a helper function if it is needed elsewhere later.